### ISSUE TITLE
Inconsistent Layout/MultilineMethodCallIndentation when adding a new method into chain

### ISSUE THREAD

**Issue by robertgauld** [rubocop-hq/rubocop#7088]
## Actual behaviour

The following code in a spec is considered fine:
```ruby
      expect(GandiV5).to receive(:get).with('https://api.gandi.net/v5/domain/domains', headers)
                                      .and_return(YAML.load_file(body_fixture))
```
However when adding a new method into the chain the Layout/MultilineMethodCallIndentation cop complains "Use 2 (not 32) spaces for indenting an expression spanning multiple lines."
```ruby
      expect(GandiV5).to receive(:get).with('https://api.gandi.net/v5/domain/domains', headers)
                                      .ordered
                                      .and_return(YAML.load_file(body_fixture))
```

## Expected behaviour

I would expect both versions to be considered fine (or not fine).

## RuboCop version

0.68.0 (using Parser 2.6.3.0, running on ruby 2.6.2 x86_64-linux)

**Comment by robertgauld** [comment]
Also in version 0.70.0 (using Parser 2.6.3.0, running on ruby 2.6.2 x86_64-linux)

**Comment by buehmann** [comment]
I cannot reproduce this with 0.73.0 and the default configuration. Could you share a `.rubocop.yml` that triggers the behaviour?

**Comment by koic** [comment]
Yeah, this issue has been resolved by #7199. Please update RuboCop to latest version.  Thanks @buehmann!

**Comment by robertgauld** [comment]
Indeed, fixed in latest version.
